hack/generate-prow-debug-kubeconfig.sh

```diff
@@ -23,27 +24,29 @@ if [ ! "$#" -eq 1 ]; then
 fi
 
 token_name="prow-debug-$1"
+token_namespace="kubevirt-prow-jobs"
```

Comment:
Would it be good to have this configurable? and default to `kubevirt-prow-jobs`

Reply:
I think we need this only for people looking at prow jobs - for now. If need be it would be easy to extend it in another PR.

Reply:
Ok sounds good to me.